### PR TITLE
1rwkk47 Improvement: validate if same day order pass like order or shipment to core

### PR DESCRIPTION
**Pull request by multinauta**
**refer:** 
https://app.clickup.com/t/1rwkk47

**Why I'm adding these lines of code:**
- to validate if same day order pass like order or shipment to core
- to create courier and shipment unit test
- to cast checkout price to float
- to change post timeout from 60 to 10 seconds

**Test:**
Screenshot from 2021-12-07 12-39-25

Screenshot from 2021-12-07 12-44-56

Screenshot from 2021-12-07 12-46-19

**Deployment:**
- enable the selector so that the same day courier is imported as an order or shipment in suite

**Message to the "nuevasfuncionalidades" channel:**
(Mejora) [1rwkk47] Validar si el courier "same day" llega a core como órden o envío según configuración del cliente.

### CODE REVIEW

**Review by davidmaogomezz — APPROVED** [review]

**Review by davidmaring7 — APPROVED** [review]

File: shipit/tags/4.2.0/readme.txt [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+Sí, con las acciones en lote, más información en el siguiente link: https://shipitcl.zendesk.com/hc/es-419/articles/360042204934--C%C3%B3mo-enviar-mis-pedidos-manualmente-desde-WooCommerce-a-Shipit-
+
+== Screenshots ==
+
+1. Credential Entry page `/ src / images / screenshot-1.png`
+2. Field cotizador messengers `/ src / images / screenshot-2.png`
+
+== Changelog ==
+= 4.2.0 =
+* Validar si compras Same Day se crean como envíos o ordenes
```
Comment (gcamposm) [inline shipit/tags/4.2.0/readme.txt:57]:
```suggestion
* Validar si compras Same Day se crean como envíos u ordenes
```

File: shipit/tags/4.2.0/readme.txt [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+* transformar precio de checkout a flotante en tarificación
```
Comment (gcamposm) [inline shipit/tags/4.2.0/readme.txt:58]: I think that in spanish is more common talk about 'decimal' in place of 'flotante'
```suggestion
* transformar precio de checkout a decimal en tarificación
```

File: shipit/trunk/readme.txt [hunk 3]
```diff
@@ -53,10 +53,14 @@ Sí, con las acciones en lote, más información en el siguiente link: https://s
 2. Field cotizador messengers `/ src / images / screenshot-2.png`
 
 == Changelog ==
+= 4.2.0 =
+* Validar si compras Same Day se crean como envíos o ordenes
+* transformar precio de checkout a flotante en tarificación
```
Comment (gcamposm) [inline shipit/trunk/readme.txt:58]: the same comments

File: shipit/trunk/shipit.php [hunk 4]
```diff
@@ -605,20 +605,29 @@ function createShipment($company, $skus, $sellerSetting, $insuranceSetting, $ord
...
   }
 }
 
+function sendOrderOrShipment($automatic_delivery, $orderPayload, $shipit, $order) {
+  if ( $automatic_delivery === false ) {
+    $integration = new Integration(get_option('shipit_user')['shipit_user'], get_option('shipit_user')['shipit_token']);
+    return $integration->massiveOrders(['orders' => [$orderPayload->build()]]);
+  } elseif ($shipit) {
+      $order = new WC_Order($order->get_id());
+      $order->add_order_note('Ya que aplicaron el metodo de retiro en tienda.');
```
Comment (gcamposm) [inline shipit/trunk/shipit.php:624]: I don't understand this message, maybe be
```suggestion
      $order->add_order_note('Ya fue aplicado el método de retiro en tienda.');
```

File: shipit/tags/4.2.0/readme.txt (hunk 1 above)
Comment (multinauta) [reply shipit/tags/4.2.0/readme.txt]: ready!

File: shipit/trunk/shipit.php (hunk 4 above)
Comment (multinauta) [reply shipit/trunk/shipit.php:624]: yes, it is because it is the continuation of another message

**Review by mgraterol — APPROVED** [review]

**Review by Barbycris — APPROVED** [review]

**Review by PamelaAlejandraGillig — APPROVED** [review]

**Review by gcamposm — APPROVED** [review]

File: shipit/trunk/shipit.php (hunk 4 above)
Comment (gcamposm) [reply shipit/trunk/shipit.php:624]: Perfect, but 'método'

**Review by HaNdlezz — APPROVED** [review]

**Review by HaNdlezz — APPROVED** [review]

**Pull request merged**